ADAL-OSX/ADAL-OSX/ADAuthenticationWebViewControllerOSX.m

```diff
@@ -125,6 +125,14 @@ - (void)webView:(WebView *)sender didFailProvisionalLoadWithError:(NSError *)err
 #pragma unused(sender)
     if ([_parentDelegate respondsToSelector:@selector(webView:didFailProvisionalLoadWithError:forFrame:)])
         [_parentDelegate webView:sender didFailProvisionalLoadWithError:error forFrame:frame];
+    
+    if (NSURLErrorCancelled == error.code)
+    {
+        //This is a common error that webview generates and could be ignored.
+        //See this thread for details: https://discussions.apple.com/thread/1727260
+        return;
+    }
+    
```

Comment:
This is already on iOS. You can find it in ADAuthenticationWebViewController:219